test/parameters.test.ts

```diff
@@ -421,7 +421,7 @@ describe('parameter handling', function () {
               description: 'ok',
               content: {
                 'application/xml': { schema: { type: 'array', items: {} } },
-                'application/json': { schema: { type: 'array', items: {} } },
+                'text/plain': { schema: { type: 'array', items: {} } },
```

Comment:
Changed this to `text/plain` because this test was already asserting that `application/xml` was the `accept` header for this. Since there aren't any JSON-compatible tiles here, `application/xml` will remain the preferred type for this case.